src/dune/build_system.ml

```diff
@@ -284,7 +278,8 @@ module Dir_triage = struct
     | Need_step2
 end
 
-module Trace : sig
+(* Stores information needed to determine if rule need to be reexecuted. *)
+module TraceDb : sig
```

Comment:
We don't `CamelCase` in dune. It should be `Trace_db`

Reply:
Fixed